dsl-reference.md

```diff
@@ -1515,9 +1515,7 @@ schema:
       petId:
         type: string
     required: [ petId ]
-from:
-  petId: '${ .pet.id }'
-to: '.petList += [ . ]'
+as: '.petList += [ . ]'
 ```
```

Comment:
There is a little mistake here I think. `output.from` became `output.as` and `output.to` became `export.as`.

Reply:
@JBBianchi Thanks for the reply.  Do you mean including both Output and Export properties in this example? Sorry, I feel like I misinterpreted something here lol.. Please let me know  

```
output:
  schema:
    format: json
    document:
      type: object
      properties:
        petId:
          type: string
      required: [ petId ]
  as:                        
    petId: '${ .pet.id }'   
export:
  as: 
   '.petList += [ . ]'  
```

Reply:
Either that or just the `output` part without `export` but I don't think it hurts to have both.
Thanks a lot ❤️

Reply:
Got it... just updated the example with both Output and Export properties.  :)